temba/channels/models.py

```diff
@@ -3600,21 +3623,19 @@ class ChannelSession(SmartModel):
                       (CANCELED, "Canceled"),
                       (INTERRUPTED, "Interrupted"),
                       (TRIGGERED, "Triggered"),
-                      (INITIATED, "Initiated"))
+                      (INITIATED, "Initiated"),
+                      (ENDING, "Ending"))
 
     external_id = models.CharField(max_length=255,
                                    help_text="The external id for this session, our twilio id usually")
     status = models.CharField(max_length=1, choices=STATUS_CHOICES, default=PENDING,
                               help_text="The status of this session")
-
     channel = models.ForeignKey('Channel',
-                                help_text="The channel that created this session")
+                                help_text="The channel that created this session", null=True)
```

Comment:
the scenario is that a user creates a ussd flow without a channel added and wants to simulate the flow